Makefile

```diff
@@ -95,19 +95,40 @@ teardown-llm-proxy:
 	@docker stop litellm-proxy
 	@docker rm litellm-proxy
 
+
+build-embeddings-arm:
+	@git clone https://github.com/huggingface/text-embeddings-inference.git
+	@docker build text-embeddings-inference -f text-embeddings-inference/Dockerfile \
+		--platform=linux/arm64 \
+		-t text-embeddings-inference-arm
+	@cd ..
+	@rm -rf text-embeddings-inference
+setup-embeddings-arm:
+	-@docker stop huggingface-embeddings
+	-@docker rm huggingface-embeddings
+	@docker system prune -f
+	@sleep 2
+	@docker run \
+		--name huggingface-embeddings \
+        -p 8080:80 \
```

Comment:
Need to document somewhere that this port needs to be set as `CUSTOM_EMBEDDINGS_ENDPOINT` before `make setup-dev` is run. Also not sure if this port works - let me know how you're doing it